distributed/core.py

```diff
@@ -312,6 +313,9 @@ def handle_comm(self, comm, shutting_down=shutting_down):
 
                 try:
                     op = msg.pop('op')
+                    if op in self.blocked_handlers:
+                        msg = "The '{op}' handler has been explicitly disallowed in {obj}, possibly due to security concerns."
+                        raise ValueError(msg.format(op=op, obj=type(self).__name__))
```

Comment:
I see that you've copied my suggestion here, foolishly assuming that I knew what I was talking about.  This will probably stop the server entirely.  We probably need to handle the error similar to how we handle errors in handlers below (look for when `handler` gets called around ten lines lower down).

One simple way to do this would be to return a handler that just errs and then reuse the stanard code below.  Dask should then route the error to the caller appropriately.

Reply:
Oh great!  Will update.

Reply:
Should I also update the "unexpected message without 'op'" `ValueError` that is raised right below mine?

Reply:
Lets leave that as is for now.  If we find that it's a concern then we'll handle it in another PR.

Reply:
With this change, the behavior is in line with what I was originally expecting -- thank you!